src/gcc.rs

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2021 IBM Corp.
+
+use crate::collector::*;
+
+const GCC_COMMAND: &'static str = "gcc -v";
+
+pub fn version(col: &mut Collector) -> Result<CollectorValue, CollectorErr> {
+    let captures = col.parse_from_command(GCC_COMMAND, r"gcc version (\d.*)", OutputStream::STDERR)?;
+    Ok(CollectorValue::Text(captures[1].to_string()))
+}
+
+pub fn flags(col: &mut Collector) -> Result<CollectorValue, CollectorErr> {
+    let captures = col.parse_from_command(GCC_COMMAND, r"Configured with: ../configure (.*)", OutputStream::STDERR)?;
```

Comment:
I ran into a panic when running because my regex did not capture the literal `Configured with: ../configure` . Not sure if this is common for everyone, but my `gcc -v` outputs `Configured with: ../src/configure`. It might be better to have this file be not specific. I was able to resolve the issue by changing the regex to `r"Configured with: [^\s]* (.*)"`, which allows for any file name (more specifically anything without whitespace).

Reply:
`r"Configured with: .*?configure (.*)"`